tasks/deployment_manager/task.ts

```diff
@@ -52,7 +52,7 @@ async function runMigration<T>(
 
   if (enact) {
     console.log('Running enactment step with artifact...', artifact);
-    await migration.actions.enact(governanceDeploymentManager, artifact);
+    await migration.actions.enact(deploymentManager, governanceDeploymentManager, artifact);
```

Comment:
most important part of this change: `Migration#enact` is now called with both the local deployment manager and the governance deployment manager